pymatgen/io/lobster/outputs.py

```diff
@@ -408,6 +409,109 @@ def icohpcollection(self):
         return self._icohpcollection
 
 
+class NciCobiList:
+    """
+    Class to read NcICOBILIST (multi-center ICOBI) files generated by LOBSTER.
+
+    Attributes:
+        is_spin_polarized (bool): Boolean to indicate if the calculation is spin polarized.
+        NciCobiList (dict): Dict containing the listfile data of the form:
+        {bond: "number_of_atoms": number of atoms involved in the multi-center interaction,
+                "ncicobi": {Spin.up: Nc-ICOBI(Ef) spin up, Spin.down: ...}},
+                "interaction_type": type of the multi-center interaction
+    """
+
+    def __init__(self, filename: str | None = None):
+        """
+        Args:
+            filename: Name of the NcICOBILIST file.
+        """
+
+        if filename is None:
+            warnings.warn("Please consider using the newest LOBSTER version (4.1.0+). See http://www.cohp.de/.")
+            filename = "NcICOBILIST.lobster"
+
+        # LOBSTER list files have an extra trailing blank line
+        # and we don't need the header.
+        with zopen(filename, "rt") as file:
+            data = file.read().split("\n")[1:-1]
+        if len(data) == 0:
+            raise OSError("NcICOBILIST file contains no data.")
+
+        # If the calculation is spin-polarized, the line in the middle
+        # of the file will be another header line.
+        self.is_spin_polarized = "spin" in data[len(data) // 2]  # TODO: adapt this for orbitalwise case
+
+        # check if orbitalwise NcICOBILIST
+        # include case when there is only one NcICOBI
+        for entry in data:  # NcICOBIs orbitalwise and non-orbitalwise can be mixed
+            if len(data) > 2 and "s]" in str(entry.split()[3:]):
+                self.orbitalwise = True
```

Comment:
Is `self.orbitalwise` missing a default value of `False`? Tests are currently failing with

```
AttributeError: 'NciCobiList' object has no attribute 'orbitalwise'
```

Reply:
Oh, I have to look more into this... :O